### DISCUSSION TITLE
Deploy Next.js to Pages is not working

### DISCUSSION THREAD

**Discussion by kennedyyung** [community/community#121057] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to deploy my website, and I didn't change any major files except basic ui changes on some of my pages and I'm getting this error with my deployment not working anymore. It's building right but when I go to my website it only shows the Read.me
![image](https://github.com/community/community/assets/97806971/e38d7672-743f-4d12-8eba-724bef674c65)

**Comment by barrownicholas** [comment]
+1 am seeing this too, `ubuntu-latest` is not getting picked up by runners
